src/app/components/faq-assistant/faq-assistant.component.ts

```diff
@@ -0,0 +1,66 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { TitleService } from '../../services/title/title.service';
+import { Title } from '../../interfaces/title/title.interface';
+import { ItemManager } from '../../item-manager/item-manager';
+
+@Component({
+  selector: 'app-faq-assistant',
+  templateUrl: './faq-assistant.component.html',
+  styleUrls: ['./faq-assistant.component.scss']
+})
+export class FaqAssistantComponent implements OnInit {
+
+  titleList: Title[];
+  answer: string;
+  items: Title[];
+  showFinish = false;
+  lastStep: boolean;
+  showBack = false;
+
+  constructor(private titleService: TitleService, private itemManager: ItemManager) { }
+
+  @Input() customClass: string;
```

Comment:
It's a property and should be declared under the constructor.

Reply:
What property?

Reply:
Sorry, yes.

Reply:
was applied